client/state/media/thunks/fetch-next-media-page.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import isFetchingNextPage from 'state/selectors/is-fetching-next-page';
+import MediaListStore from 'lib/media/list-store';
+import { dispatchFluxFetchMediaItems } from 'state/media/utils/flux-adapter';
+import { requestMedia } from 'state/media/actions';
+
+const debug = debugFactory( 'calypso:media' );
+
+/**
+ * Redux thunk to fetch next page of media items
+ *
+ * @param {number} siteId site identifier
+ */
+export const fetchNextMediaPage = ( siteId ) => ( dispatch, getState ) => {
+	if ( isFetchingNextPage( getState(), siteId ) ) {
```

Comment:
I think I'm missing something, but where does `nextPage` get set to true? Wouldn't we want to do that right after this if/return?

Reply:
We do, the [`fetching` reducer](https://github.com/Automattic/wp-calypso/blob/master/client/state/media/reducer.js#L443) sets it to `true` right after dispatching `requestMedia` a few lines below.

Reply:
Oh my gosh, duh, I should have remembered that/looked it up 🤦‍♀️ Thanks for reminding me!

Reply:
No worries :)